### ISSUE TITLE
Client prediction input pipeline: use of Redis and side-effects

### ISSUE THREAD

**Issue by mkaranasou** [deflect-ca/baskerville#18]
Let's list all the possible issues we might face if we use redis to store all data or store only id_client, id_group:

- [ ] Too much traffic could throw redis off and we might end up losing all the data - so we'll have incoming predictions and nothing to match them with and store in the db.

- [ ] Predictions may come in slower than expected, so data could expire (do we want to lose all data, or could a client resend stuff afterwards to predict off-line?)

- [ ] ..

Basically, can we afford to lose data?
